contributing.md

```diff
@@ -21,7 +21,40 @@ Please create an issue in this repo if it directly concerns contents within this
 - Improving workflows within this repo
 - Adding documentation
 
-### Creating Pull Requests
+## Creating Pull Requests
+
+### General guide of code contributions
+
+1. Create a personal fork of the project on Github.
+2. Clone the fork on your local machine. Your remote repo on Github is called `origin`.
+
+   ```sh
+   git clone git@github.com:USERNAME/REPOSITORY_NAME.git
+   ```
+
+3. Add the original repository as a remote called `upstream`.
+
+   ```sh
+   git remote add upstream git@github.com:operate-first/REPOSITORY_NAME.git
+   ```
+
+4. If you created your fork a while ago be sure to pull upstream changes into your local repository.
+
+   ```sh
+   git checkout master # or main
+   git fetch upstream
+   git rebase upstream/master  # or upstream/main
+   git push origin master
+   ```
+
+5. Create a new branch to work on. Create this branch from the default, up to date branch like `master` or `main`.
+6. Install `pre-commit` git hook.
```

Comment:
Could we change it to say "Install `pre-commit` git hook by running `pre-commit install`

Reply:
We certainly can! :+1: